roles/arvados/tasks/main.yaml

```diff
@@ -12,7 +12,7 @@
 # so unfortunately for RPM distributions like CentOS this needs to be done manually
 - name: Register the Curoverse signing key in RPM’s database
   shell: >
-    gpg --keyserver pool.sks-keyservers.net --recv-keys 1078ECD7 &&
+    gpg --keyserver pgp.mit.edu --recv-keys 1078ECD7 &&
```

Comment:
The pgp pool contains plenty of unreliable pgp hosts that timeout. During my testing it was failing like 70% of the time. pgp.mit.edu might not be the best, but it has been holding up for a long time for me.